bucket/deew.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "3.0.1",
+    "description": "Dolby Encoding Engine Wrapper",
+    "homepage": "https://github.com/pcroland/deew",
+    "license": "MIT License",

```

```suggestion
    "license": "MIT",
```
Use SPDX License identifier.